exchanges/kraken/kraken_websocket.go

```diff
@@ -540,17 +542,30 @@ func (k *Kraken) wsProcessTrades(response []any, pair currency.Pair) error {
 		if !ok {
 			return errors.New("unidentified trade data received")
 		}
-		timeData, err := strconv.ParseFloat(t[2].(string), 64)
+		if len(t) < 4 {
+			return fmt.Errorf("unexpected trade data length: %s", t)
+		}
+		ts, ok := t[2].(string)
+		if !ok {
+			return common.GetTypeAssertError("string", t[2], "timeData")
```

Comment:
Nits:
* Think we need to prefix the name here, so `trade.*` for all these
* Names should match api docs, so `.time`
```suggestion
			return common.GetTypeAssertError("string", t[2], "trade.time")
```